homeassistant/components/qbus/light.py

```diff
@@ -0,0 +1,108 @@
+"""Support for Qbus light."""
+
+from typing import Any
+
+from qbusmqttapi.discovery import QbusMqttOutput
+from qbusmqttapi.state import QbusMqttAnalogState, StateType
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, ColorMode, LightEntity
+from homeassistant.components.mqtt import ReceiveMessage
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util.color import brightness_to_value, value_to_brightness
+
+from .coordinator import QbusConfigEntry
+from .entity import QbusEntity, add_new_outputs
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: QbusConfigEntry, add_entities: AddEntitiesCallback
```

Comment:
```suggestion
    hass: HomeAssistant, entry: QbusConfigEntry, async_add_entities: AddEntitiesCallback
```
Please call it `async_add_entities`